com.chartboost.mediation/Documentation/integration/loading-ads.md

```diff
@@ -57,15 +57,43 @@ Debug.Log($"Fullscreen Placement Loaded with PlacementName: {placementName}")
+- Publishers can choose whether to use Adaptive Ads or Fixed Ads in a given placement
+- Fixed Ads are supported in Adaptive Ad placements (backwards compatible)
+- Publishers should know whether an ad is fixed or flexible and receive the dimensions of fixed ads
+- Publishers can align the ad horizontally and/or vertically
+- Publishers can resize the ad container to fit the ad or optionally discard oversized ads that are rendered in the container.
+- The ad container can be in-line or on top of publisher content.
+
+To use this new ad format, Publishers will need to create a new Adaptive Banner placement in their platform and integrate with the new Adaptive Banner APIs.
+
+We have added a new Banner API `ChartboostMedaitionBannerView` to allow usage of adaptive banners. The previous `ChartboostMedaitionBannerAd` API has now been deprecated.
 
-Since Chartboost Mediation Unity SDK 4.6.X we have deprecated the previous approach for loading banner placements. A new Banner API `ChartboostMedaitionBannerView` has been provided which makes use of C# asycn/await methods in order to await for load and show and also provides support to load adaptive size banners.
 
 Another API `ChartboostMediationUnityBannerAd` is also provided which allows usage of unity gameobjects to load a banner ad within it.
 
+The API class `ChartboostMediationBannerSize` will support both fixed and adaptive banners:
+
+|Field|Description|
+|--|--|
+|`static ChartboostMediationBannerSize Standard`|Static constant that returns a fixed `STANDARD` banner with a size of 320x50.|
+|`static ChartboostMediationBannerSize MediumRect`|Static constant that returns a fixed MREC `MEDIUM` banner with a size of 300x250.|
+|`static ChartboostMediationBannerSize Leaderboard`|Static constant that returns a fixed `LEADERBOARD` banner with a size of 728x90.|
+|`ChartboostMediationBannerType BannerType`|An enum specifying that the ad size is fixed (size cannot change), or adaptive (size can change but must maintain aspectRatio). This is an integer based enum.|
+|`float Width`|The width of the ad.|
+|`float Height`|The height of the ad.|
+|`float AspectRatio`|The aspect ratio of the ad. This can be derived from the size. Will be 0 if either the width or the height are <= 0.|
+|`static ChartboostMediationBannerSize Adaptive(float width)`|Creates a flexible/adaptive banner size with 0 height.|
+|`static ChartboostMediationBannerSize Adaptive(float width, float height)`|Creates a flexible/adaptive banner size with a width and max height. Used either when the height of an inline ad should be capped, or when requesting an anchored banner. Could alternatively take a size instead of width/maxHeight, however width/maxHeight mimics Google’s methods.|
```

Comment:
Remove line: "Could alternatively take a size instead of width/maxHeight, however width/maxHeight mimics Google’s methods."